### PR TITLE
Feature/MapKit extensions

### PR DESCRIPTION
**Pull request by borut-t**
Added some useful MapKit extensions.

### CODE REVIEW

File: Sources/Extensions/MapKit/MKMapView+PovioKit.swift [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+//
+//  Created by Borut Tomažin on 11/11/2020.
+//  Copyright © 2020 Povio Labs. All rights reserved.
+//
+
+import MapKit.MKMapView
+
+public extension MKMapView {
+  /// Current zoom level
+  var zoom: Double {
```
Comment (nejcvivod) [inline Sources/Extensions/MapKit/MKMapView+PovioKit.swift:13]: What does this denote, percent?

File: Sources/Extensions/MapKit/MKMapView+PovioKit.swift [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+    log2(360 * ((Double(frame.size.width) / 128) / region.span.longitudeDelta))
+  }
+  
+  /// Returns radius distance in meters from map center to the top of the visible screen
+  var visibleRadius: Double {
+    let centerLocation = CLLocation(latitude: centerCoordinate.latitude, longitude: centerCoordinate.longitude)
+    let topCenterCoordinate = convert(.init(x: bounds.size.width / 2.0, y: 0), toCoordinateFrom: self)
```
Comment (nejcvivod) [inline Sources/Extensions/MapKit/MKMapView+PovioKit.swift:20]: We could use bounds.midX instead of bounds.size.width / 2.0 🤔

**Review by nejcvivod — APPROVED** [review]
👍 looks great

File: Sources/Extensions/MapKit/MKMapView+PovioKit.swift (hunk 1 above)
Comment (borut-t) [reply Sources/Extensions/MapKit/MKMapView+PovioKit.swift:13]: Good question. This was taken out of context, therefore not good enough to be put here. I'll remove it.

File: Sources/Extensions/MapKit/MKMapView+PovioKit.swift (hunk 2 above)
Comment (borut-t) [reply Sources/Extensions/MapKit/MKMapView+PovioKit.swift:20]: Good point 👌

**Pull request merged**